test/expressionrouter/suite_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build expression_router_tests
+// +build expression_router_tests
+
+package expressionrouter
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/blang/semver/v4"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/metallb"
+	"github.com/kong/kubernetes-testing-framework/pkg/environments"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/test/internal/helpers"
+	"github.com/kong/kubernetes-ingress-controller/v2/test/internal/testenv"
+)
+
+var (
+	env environments.Environment
+	ctx context.Context
+)
+
+func TestMain(m *testing.M) {
```

Comment:
What's specifically the reason we need a separate test package for this? To configure kong (via ktf addon) with a new router flavor which is not being done in standard integration tests suite?

Reply:
The reason why I create a separate test suite:
- The translation to expression routes are still working in progress and I want to test the function without integrating the function into controller in the early phase
- The integration test suite will run a ingress controller in its `TestMain`, which will modify the Kong configuration from k8s objects. This configuration may possibly override the Kong configuration I used in my test cases. So The tests could NOT be run together with the integration test suite.
When We finished the implementation of translating k8s objects into Kong configs, we will add integration tests for them. In that phase, we may delete the separate test suite.